service/src/test/java/bio/terra/user/app/controller/PublicApiControllerTest.java

```diff
@@ -0,0 +1,75 @@
+package bio.terra.user.app.controller;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.redirectedUrl;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import bio.terra.user.app.configuration.VersionConfiguration;
+import bio.terra.user.service.status.StatusService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = PublicApiController.class)
+@WebMvcTest
+class PublicApiControllerTest {
```

Comment:
On line 34, you say "when status endpoint is called, return status". Then on line 35, you say "call status endpoint; check that status is returned. IIUC, you're not testing any code in this repo.

Let's copy WSM WorkspaceApiControllerTest.java.

- BaseTest.java has:
```
// Configure MockMvc not to print additional debugging information. Otherwise, this will print out
// request headers including test user access tokens, which should not be written to test output.
@AutoConfigureMockMvc(print = MockMvcPrint.NONE)
```
I see you put `@AutoConfigureMockMvc` in AppTest. Can we delete AppTest and just have BaseTest?
- Have this file extend BaseTest. (Future PRs can add BaseUnitTest, BaseConnectedTest.)
- WorkspaceApiControllerTest.java does not have any annotations. I'm not too clear on @AutoConfigureMockMvc vs @WebMvcTest, but they appear to be different.
- WorkspaceApiControllerTest.java is testing workspace controller/service. There is no `@MockBean mockWorkspaceService`.  (samService is mocked because for unit tests, we don't want to make calls to real sam.) So for this file, don't have `@MockBean private StatusService statusService;`. Then the real StatusService will be used, which is what you want.